.env.example

```
@@ -56,21 +56,20 @@ BAAI_EMBEDDING_MODEL="BAAI/bge-m3"
-# query expansion
-QUERY_EXPANSION_MODEL="google/gemini-2.5-flash-lite"
+QUERY_EXPANSION_MODEL="openrouter/google/gemini-2.5-flash"
 
 # reranking model
 BAAI_RERANKER_MODEL="BAAI/bge-reranker-v2-m3"
 
 # qdrant
-QDRANT_HOST="localhost"
+QDRANT_HOST=localhost
 QDRANT_PORT=6333
 QDRANT_API_KEY=ENTER_YOUR_PREFERRED_PASSWORD
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The QDRANT_API_KEY key should go before the QDRANT_HOST key

(UnorderedKey)